lib/Drupal/Generic/Drupal7.php

```diff
@@ -0,0 +1,17 @@
+<?php
+
+/**
+ * @file
+ * Contains \Drupal\Generic\Drupal7.
+ */
+
+namespace Drupal\Generic;
+
+class Drupal7 {
+  public function __call($method, $args) {
+    if (function_exists('drupal_' . $method)) {
+      return call_user_func_array('drupal_' . $method, $args);
+    }
+    return call_user_func_array($method, $args);
```

Comment:
Can we get some tests for that :)?

Will need to be an integration test with a test module as unit testing of methods is not possible - as far as my experience goes.